docs/learn/encyclopedia/data-format/xdr.mdx

```diff
@@ -2,6 +2,9 @@
 title: XDR
 ---
 
+import { CodeExample } from "@site/src/components/CodeExample";
+import { Alert } from "@site/src/components/Alert";
```

Comment:
I don't think you're actually using the `<Alert>` component anywhere? Unless I'm mistaken, let's remove it from the imports.

Reply:
Likely just a copy-paste mishap from another page! I'll drop it